### ISSUE TITLE
Export CSS2DRenderer in jsm

### ISSUE THREAD

**Issue by linonetwo** [mrdoob/three.js#16337]
Currently it can:
```js
import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls';
```
But can't
```js
import { CSS2DRenderer, CSS2DObject } from 'three/examples/js/renderers/CSS2DRenderer';
```

Can you export `/renderer` to `/jsm/`?

**Comment by linonetwo** [comment]
I've refactor it to ts, paste here for reference.

CSS2DObject.ts
```ts
/**
 * @author mrdoob / http://mrdoob.com/
 */
import * as THREE from 'three';

export default class CSS2DObject extends THREE.Object3D {
  public element: HTMLElement;

  constructor(element: HTMLElement) {
    super();

    this.element = element;
    this.element.style.position = 'absolute';

    this.addEventListener('removed', function() {
      if (this.element.parentNode !== null) {
        this.element.parentNode.removeChild(this.element);
      }
    });
  }
}
```

CSS2DRenderer.ts
```ts
/**
 * @author mrdoob / http://mrdoob.com/
 */
import * as THREE from 'three';

import CSS2DObject from './CSS2DObject';

export default class CSS2DRenderer {
  public domElement = document.createElement('div');
  private width: number;
  private height: number;
  private widthHalf: number;
  private heightHalf: number;
  private viewMatrix = new THREE.Matrix4();
  private viewProjectionMatrix = new THREE.Matrix4();
  private vector = new THREE.Vector3();
  private cache = {
    objects: new WeakMap(),
  };
  private getDistanceToSquaredA = new THREE.Vector3();
  private getDistanceToSquaredB = new THREE.Vector3();

  constructor() {
    this.domElement.style.overflow = 'hidden';
  }

  public zOrder(scene: THREE.Scene) {
    const sorted = this.filterAndFlatten(scene).sort(function(a, b) {
      const distanceA = this.cache.objects.get(a).distanceToCameraSquared;
      const distanceB = this.cache.objects.get(b).distanceToCameraSquared;

      return distanceA - distanceB;
    });

    const zMax = sorted.length;

    for (let i = 0, l = sorted.length; i < l; i++) {
      sorted[i].element.style.zIndex = String(zMax - i);
    }
  }
  public getDistanceToSquared(object1: THREE.Object3D, object2: THREE.Object3D) {
    this.getDistanceToSquaredA.setFromMatrixPosition(object1.matrixWorld);
    this.getDistanceToSquaredB.setFromMatrixPosition(object2.matrixWorld);

    return this.getDistanceToSquaredA.distanceToSquared(this.getDistanceToSquaredB);
  }

  public render(scene: THREE.Scene, camera: THREE.Camera) {
    scene.updateMatrixWorld();

    if (camera.parent === null) {
      camera.updateMatrixWorld(false);
    }

    this.viewMatrix.copy(camera.matrixWorldInverse);
    this.viewProjectionMatrix.multiplyMatrices(camera.projectionMatrix, this.viewMatrix);

    this.renderObject(scene, camera);
    this.zOrder(scene);
  }

  public getSize() {
    return {
      width: this.width,
      height: this.height,
    };
  }

  public setSize(width: number, height: number) {
    this.width = width;
    this.height = height;

    this.widthHalf = this.width / 2;
    this.heightHalf = this.height / 2;

    this.domElement.style.width = width + 'px';
    this.domElement.style.height = height + 'px';
  }
  private filterAndFlatten(scene: THREE.Scene) {
    const result: CSS2DObject[] = [];

    scene.traverse(object => {
      if (object instanceof CSS2DObject) {
        result.push(object);
      }
    });

    return result;
  }

  private renderObject(object: THREE.Object3D, camera: THREE.Camera) {
    if (object instanceof CSS2DObject) {
      this.vector.setFromMatrixPosition(object.matrixWorld);
      this.vector.applyMatrix4(this.viewProjectionMatrix);

      const element = object.element;
      const style =
        'translate(-50%,-50%) translate(' +
        (this.vector.x * this.widthHalf + this.widthHalf) +
        'px,' +
        (-this.vector.y * this.heightHalf + this.heightHalf) +
        'px)';

      element.style.transform = style;

      element.style.display = object.visible && this.vector.z >= -1 && this.vector.z <= 1 ? '' : 'none';

      const objectData = {
        distanceToCameraSquared: this.getDistanceToSquared(camera, object),
      };

      this.cache.objects.set(object, objectData);

      if (element.parentNode !== this.domElement) {
        this.domElement.appendChild(element);
      }
    }

    for (let i = 0, l = object.children.length; i < l; i++) {
      this.renderObject(object.children[i], camera);
    }
  }
}
```

**Comment by Mugen87** [comment]
Ideally, we should first try to convert the class with the [modularize.js](https://github.com/mrdoob/three.js/blob/dev/utils/modularize.js).

**Comment by Mugen87** [comment]
Solved via #16472
